packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -672,6 +672,335 @@ Terra.describeViewports('Date Picker', ['medium'], () => {
+          browser.keys('t');
+
+          const currentDate = new Date();
+          expect($('input')).toHaveValue(currentDate.toISOString().split('T')[0]);
+        });
+
+        it('sets Date to today on day input', () => {
+          Terra.hideInputCaret('input[name="terra-date-day-date-input"]');
+          $('input[name="terra-date-day-date-input"]').click();
+          browser.keys('20');
+          $('input[name="terra-date-day-date-input"]').click();
+          browser.keys('t');
+
+          const currentDate = new Date();
+          expect($('input')).toHaveValue(currentDate.toISOString().split('T')[0]);
+        });
+
+        it('sets Date to today on year input', () => {
+          Terra.hideInputCaret('input[name="terra-date-year-date-input"]');
+          $('input[name="terra-date-year-date-input"]').click();
+          browser.keys('2005');
+          $('input[name="terra-date-year-date-input"]').click();
+          browser.keys('t');
+
+          const currentDate = new Date();
+          expect($('input')).toHaveValue(currentDate.toISOString().split('T')[0]);
+        });
+      });
+
+      describe('if a valid date', () => {
+        beforeEach(() => {
+          browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-start-date');
+        });
+
+        it('sets Date to today on month input', () => {
+          Terra.hideInputCaret('input[name="terra-date-month-date-input"]');
+          $('input[name="terra-date-month-date-input"]').click();
+          browser.keys('t');
+
+          const currentDate = new Date();
+          expect($('input')).toHaveValue(currentDate.toISOString().split('T')[0]);
+        });
+
+        it('sets Date to today on day input', () => {
+          Terra.hideInputCaret('input[name="terra-date-day-date-input"]');
+          $('input[name="terra-date-day-date-input"]').click();
+          browser.keys('t');
+
+          const currentDate = new Date();
+          expect($('input')).toHaveValue(currentDate.toISOString().split('T')[0]);
+        });
+
+        it('sets Date to today on year input', () => {
+          Terra.hideInputCaret('input[name="terra-date-year-date-input"]');
+          $('input[name="terra-date-year-date-input"]').click();
+          browser.keys('t');
+
+          const currentDate = new Date();
+          expect($('input')).toHaveValue(currentDate.toISOString().split('T')[0]);
+        });
+      });
+    });
+
+    describe('-', () => {
+      describe('if blank', () => {
+        beforeEach(() => {
+          browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-default');
+        });
+
+        it('sets Date to yesterday on month input', () => {
+          Terra.hideInputCaret('input[name="terra-date-month-date-input"]');
+          $('input[name="terra-date-month-date-input"]').click();
+          browser.keys('-');
+
+          const currentDate = new Date();
+          currentDate.setDate(currentDate.getDate() - 1);
```

Comment:
I find this variable naming confusing.

Reply:
yea i can easily go through and on the `-` and `+` shortcut tests rename to yesterday and tomorrow for the variable names

Reply:
Changed here https://github.com/cerner/terra-framework/pull/1507/commits/88f3dab1071c7e5c7e94082247ddafb878b2631d